src/actions/twoFactor.js

```diff
@@ -75,8 +75,11 @@ export const checkTwoFactorCode = (code, mode = AUTH_MODE.TWO_FA_MAIL) => {
       auth_mode: mode
     })
       .then(() => {
-        dispatch({ type: CHECK_TWO_FACTOR_CODE_SUCCESS })
-        dispatch(updateInfo('two_fa', true))
+        dispatch({
+          type: CHECK_TWO_FACTOR_CODE_SUCCESS,
+          field: 'auth_mode',
```

Comment:
Is `auth_mode` a new field ? If so I think we should use `authMode` instead to be compliant with recent guidelines.

Reply:
This is a field based directly on the stack name, it's why this is not camelCase here